packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.js

```diff
@@ -68,13 +81,25 @@ export default class QuanticNotifications extends LightningElement {
     );
   }
 
-  get notifications() {
-    return (
-      this.notifyTriggerState?.notifications.map((notification, index) => ({
-        value: notification,
-        id: index,
-      })) ?? []
-    );
+  handleSearchStatusChange() {
+    if (!this.searchStatus.state.isLoading) {
+      this.notifications =
+        this.notifyTrigger?.state?.notifications.map((notification, index) => ({
+          value: notification,
+          id: index.toString(),
+          visible: true,
+        })) ?? [];
+    }
```

Comment:
I would do that, that way on each "new query" the notifications are actually cleared so for example they don't display while the rest of the UI is in "loading" mode, aka showing placeholders.

```suggestion
    if (this.searchStatus?.state?.isLoading) {
      this.notifications = [];
    } else {
      this.notifications =
        this.notifyTrigger?.state?.notifications.map((notification, index) => ({
          value: notification,
          id: index.toString(),
          visible: true,
        })) ?? [];
    }
```

Also why did you add the `toString()` ?

Reply:
I will test the first point you bring up, seems to make sense

for the second question, it is because in the `handleNotificationClose` when we do: `if (notification.id === currentNotificationId)`, the currentNotificationId is of type string, so it was so that they could do the comparison, but when I think about it, I could probably just do `if (notification.id.toString() === currentNotificationId)`